src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -673,6 +684,15 @@ bool QMCFixedSampleLinearOptimizeBatched::processOptXML(xmlNodePtr opt_xml,
   block_lmStr = lowerCase(block_lmStr);
   block_lm    = (block_lmStr == "yes");
 
+  filter_paramStr = lowerCase(filter_paramStr);
+  filter_param_   = (filter_paramStr == "yes");
+
+  filter_infoStr = lowerCase(filter_infoStr);
+  filter_info_   = (filter_infoStr == "yes");
+
+  store_samplesStr = lowerCase(store_samplesStr);
+  store_samples_   = (store_samplesStr == "yes");
```

Comment:
Why not m_param.add directly filter_param_, filter_info_, store_samples_?

Reply:
I find when I try m_param.add directly for those booleans, the user input must be "1" to for them to be set to true. Using the strings seemed to allow for more intuitive user input and follows what's done for the blocked LM and excited state settings.

Reply:
Thanks. I will keep a note to update bool handling.

Reply:
I actually fixed the ParameterSet handling boolean in the develop.
@PDoakORNL seems out of office. As we are waiting for his review, could you try removing the input strings?

Reply:
Ok, thanks. Removing the strings works now and I've pushed a commit.